### ISSUE TITLE
`<main>` is missing for semantic HTML across pages

### ISSUE THREAD

**Issue by benaltair** [bahaistudies/jbs#54]
![image](https://user-images.githubusercontent.com/17771229/116310110-74023c00-a75e-11eb-937f-12b5e821716e.png)
